docs/content/resources.md

```diff
@@ -9,9 +9,12 @@ menu:
 
 ---
 
-Converge has resources that you can use to make various changes on your system.
-For example, `file.content` will control the content of a file, and `file.mode`
-will control that file's mode.
+Converge has resources you can use to make various changes on your system. For
+example, `file.content` will control the content of a file, and `file.mode`
+will control a file's mode.
```

Comment:
What about saying "`file.content` controls what's in a file, and `file.mode` sets its permissions." instead of repeating "content" and "mode"?